### ISSUE TITLE
Is it possible to achieve animation during drag-n-move?

### ISSUE THREAD

**Issue by yccheok** [DustinBryant/WpfDashboardControl#2]
Hi @DustinBryant 

Thank you for open sourcing WpfDashboardControl. It provides a good foundation, for me to create a card-liked dashboard UI.

Currently, I'm planning to port the following Android app to desktop app.

I wish to achieve the following animation during drag-n-move - https://youtu.be/aRvM64dBNA0

May I know, is it possible achieve animation during drag-n-move, by modifying your WpfDashboardControl? Can you provide me some hint on where I should start from?

Thanks.

**Comment by DustinBryant** [comment]
Yes, it is possible to do drag-n-move animations in WPF. These are primarily done with storyboards in WPF. I've only done basic storyboards in WPF so I wouldn't be able to provide much assistance in that implementation.

Now, if your "widgets" are all going to be the same size, maybe look at these alternatives:
https://github.com/rulyotano/wpf-drag-animated-panel
https://github.com/ramer/PanelDragDropReorder

I believe the first one (wpf-drag-animated-panel) leans more towards what you are looking for. If the "widgets" are going to be different sizes then you'll probably want to use mine.

There's a few ways you could accomplish this. I'd suggest intercepting the move on this file:
https://github.com/DustinBryant/WpfDashboardControl/blob/master/WpfDashboardControl/Resources/DashboardControl/DashboardHost.xaml.cs
 
And this method within that file: SetWidgetRowAndColumn()

That's what does the moving of widgets around. I'd also suggest not doing the animation if the provided WidgetHost is the _draggingHost. That way your animations will only occur on actual moving widgets.

Hope this was enough information, if you have any other questions let me know and I can point you to the right direction.

**Comment by yccheok** [comment]
Thank you very much for the pointer. I will check them out.
